app/controllers/items/staff_conversation.js

```diff
@@ -0,0 +1,17 @@
+import Ember from "ember";
+import MessageBase from "stock/mixins/messages_base";
+
+export default Ember.Controller.extend(MessageBase, {
+  isPrivate: true,
+
+  canManageItemMessages: Ember.computed.alias(
+    "session.currentUser.canManageItemMessages"
+  ),
+
+  actions: {
+    sendMessage() {
+      let values = this.prepareMessageObject("Package");
+      this.createMessage(values);
```

Comment:
There doesn't seem to be any error management here

Reply:
Yes, it is handled here https://github.com/crossroads/stock.goodcity/pull/805/files#diff-ed05eabf206ef70eacdb8b04b22c229fR58

Reply:
But we would never know something went wrong here. Other than seeing an uncaught promise in the console (cause it throws the error).

You should probably `return` the promise, catch the error and display an error message. (you can use async mixin for that)